ci/jenkins/test-vmc.sh

```diff
@@ -448,11 +449,20 @@ function run_integration {
     VM_IP=$(govc vm.ip ${VM_NAME}) # wait for VM to be on
 
     set -x
-    echo "===== Run Integration tests ====="
-    # umask ensures that files are cloned with the correct permissions so that Docker caching can be leveraged
-    ${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "umask 0022 && git clone ${ghprbAuthorRepoGitUrl} antrea && cd antrea && git checkout ${GIT_BRANCH} && DOCKER_REGISTRY=${DOCKER_REGISTRY} ./build/images/ovs/build.sh --pull && NO_PULL=${NO_PULL} make docker-test-integration"
-    if [[ "$COVERAGE" == true ]]; then
+    if [[ ${flag} == "multicluster" ]];then
+      echo "===== Run Multi-cluster Integration tests ====="
+      # umask ensures that files are cloned with the correct permissions so that Docker caching can be leveraged
+      ${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "umask 0022 && git clone ${ghprbAuthorRepoGitUrl} antrea && cd antrea && git checkout ${GIT_BRANCH} && cd multicluster && make test-integration"
```

Comment:
Please correct me if I'm wrong since I'm not so familiar with the multi-cluster feature:
1. Shouldn't this PR be merged into `feature/multi-cluster` branch instead of `main`?
2. I didn't find `test-integration` in `feature/multi-cluster` branch's `multicluster/Makefile` file. So I am also not sure if `./build/images/ovs/build.sh` is necessary like the usual integration test.

Reply:
Hi @lzhecheng for the first question, can we try `test-mc-integration` if we merge this PR to `feature/multi-cluster` branch? for the second one, MC integration test doesn't depend on ovs related, it's purely CRD manipulation, so I think it should be fine. we will verify connection in e2e codes.

Reply:
Discussed offline. This PR will be splitted into 2 PRs on main and feature branch.